t/quri.lisp

```diff
@@ -7,27 +7,51 @@
 
 (plan nil)
 
+(defun common-assertions-uri-equivalences ()
+  (is (uri "http://test/foo")
+      (make-uri :scheme "http" :host "test" :path "/foo")
+      "Same scheme, host, port, path query and fragment.")
+  (isnt (uri "http://test")
+        (make-uri :scheme "https" :host "test")
+        "Differ by scheme.")
+  (isnt (uri "http://test")
+        (make-uri :scheme "http" :host "test" :port 81)
+        "Differ by port.")
+  (isnt (uri "http://test/")
+        (make-uri :scheme "http" :host "test" :path "/" :query "?")
+        "Differ by query.")
+  (isnt (uri "http://test/#foo")
+        (make-uri :scheme "http" :host "test" :path "/" :fragment "bar")
+        "Differ by fragment.")
+  (is (uri "http://test")
+      (make-uri :scheme "http" :host "test" :path "")
+      "The NIL and empty string path are equivalent."))
+
 (subtest "uri="
-  (let ((prove:*default-test-function* #'uri-equal))
-    (is (uri "http://b.hatena.ne.jp")
-        (uri "http://b.hatena.ne.jp:80"))
```

Comment:
You are no longer testing this.

Reply:
You are also no longer testing with a URL that has seperate TLD, domain _and_ host.
I suggest you revert back to the original URL.  If you think we should test against a URL like `http://test`, it's OK to add it I think, but it does not replace the original test.

Reply:
Good point. I'll address it.